Packages/com.unity.inputsystem/InputSystem/InputManager.cs

```diff
@@ -1905,7 +1919,8 @@ internal unsafe void OnUpdate(InputUpdateType updateType, int eventCount, IntPtr
             var buffersToUseForUpdate = updateType;
 #if UNITY_EDITOR
             gameIsPlayingAndHasFocus = InputConfiguration.LockInputToGame ||
-                (UnityEditor.EditorApplication.isPlaying && Application.isFocused);
+                (UnityEditor.EditorApplication.isPlaying && Application.isFocused) ||
+                ((updateMask & (InputUpdateType)(1 << 31)) != 0);
```

Comment:
Now realize this won't work correctly. What it will effectively do is take input away from the editor (like `LockInputToGame` does) whereas the intention would be that the input is visible in both when `runInBackground` is enabled. With the current event processing, that'd take more extensive changes. Think for now I'll just leave it be and go back to how it was before.